src/navigation.vue

```diff
@@ -140,6 +140,17 @@ export default {
 		return {
 		}
 	},
+	computed: {
+		editorUrl() {
+			if (this.$store.state.metadata.id) {
+				return "/dataset/" + this.$store.state.metadata.id
```

Comment:
is this a valid route as we remove this below?

Reply:
I'm not sure what you mean. The /dataset/:id route still exists, I just removed the default child route that used to redirect it to an undefined tab and caused the url to be shown as empty.

Reply:
ok